rest/src/main/groovy/whelk/rest/api/SearchUtils2.java

```diff
@@ -27,10 +27,12 @@
 public class SearchUtils2 {
     private final QueryUtil queryUtil;
     private final Disambiguate disambiguate;
+    private final JsonLd jsonLd;
 
     SearchUtils2(Whelk whelk) {
         this.queryUtil = new QueryUtil(whelk);
         this.disambiguate = new Disambiguate(whelk);
+        this.jsonLd = whelk.getJsonld();
```

Comment:
No need to pass on the JsonLd object, use `Disambiguate.isSubclassOf()` instead.